.github/workflows/test.yml

```
@@ -0,0 +1,80 @@
+
+jobs:
+  changes:
+    runs-on: ubuntu-latest
+    outputs:
+      kv-parser: ${{ steps.filter.outputs.kv-parser }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Check for changes
+        uses: dorny/paths-filter@v2
```

actionlint (1.7.7)

the runner of "dorny/paths-filter@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)